src/ts/Detail/api.ts

```diff
@@ -1,7 +1,7 @@
 import API from '../API';
 
 export function get(id) {
-  return fetch(`${API.offer}/${id}`, {
+  return window['fetch'](`${API.offer}/${id}`, {
```

Comment:
I use api requests after user action or in `componentDidMount` for now, so it's not invoked on the server side.